book/src/guides/indent.md

```diff
@@ -1,35 +1,33 @@
 # Adding Indent Queries
 
-Helix uses tree-sitter to correctly indent new lines. This requires
-a tree-sitter grammar and an `indent.scm` query file placed in
-`runtime/queries/{language}/indents.scm`. The indentation for a line
-is calculated by traversing the syntax tree from the lowest node at the
-beginning of the new line. Each of these nodes contributes to the total
-indent when it is captured by the query (in what way depends on the name
-of the capture).
-
-Note that it matters where these added indents begin. For example,
-multiple indent level increases that start on the same line only increase
-the total indent level by 1.
+Helix uses tree-sitter to correctly indent new lines. This requires a
+tree-sitter grammar and an `indent.scm` query file placed in
+`runtime/queries/{language}/indents.scm`. The indentation for a line is
+calculated by traversing the syntax tree from the lowest node at the beginning
+of the new line. Each of these nodes contributes to the total indent when it is
+captured by the query (in what way depends on the name of the capture).
+
+Note that it matters where these added indents begin. For example, multiple
+indent level increases that start on the same line only increase the total
+indent level by 1.
 
 ## Scopes
 
-Added indents don't always apply to the whole node. For example, in most
-cases when a node should be indented, we actually only want everything
-except for its first line to be indented. For this, there are several
-scopes (more scopes may be added in the future if required):
+Added indents don't always apply to the whole node. For example, in most cases
```

Comment:
There seems to be a lot of wrapping changes even though the content didn't change at all, which makes the diff harder to review since it's difficult to spot what actually changed.

Reply:
It varies a lot from file to file, in some files the problem is hardly there. I don't know what I can do about it. I could temporarily format all the files to a requested line length in order to fit better with the original, but doing so may cause the problem to appear in other files that were not affected by it before.

The best solution would be to format the original files using the same settings:

```
[[language]]
name = "markdown"
formatter = { command = 'prettier', args = [
  "--parser",
  "markdown",
  "--prose-wrap",
  "always",       # <always|never|preserve>
] }
```

Reply:
It does, but it displays the diffs inline so it can be hard to see what changed. It's also impossible to leave comments without switching back to the plain text mode

Reply:
I have reopened https://github.com/helix-editor/helix/pull/5545 to format the original documentation, what do you think of that idea?